docs/pages/distribution/publishing-websites-webpack.mdx

```diff
@@ -0,0 +1,337 @@
+---
+title: Publish websites with Webpack
+description: Different ways to publish the Expo web app with third-party services.
+---
+
+import { APIBox } from '~/components/plugins/APIBox';
+import { Step } from '~/ui/components/Step';
+import { Tab, Tabs, TabsGroup } from '~/ui/components/Tabs';
+import { Table, TableHead, Row, HeaderCell } from '~/ui/components/Table';
+import { Terminal } from '~/ui/components/Snippet';
+import { YesIcon, NoIcon, PendingIcon } from '~/ui/components/DocIcons';
+
+> **warning** In SDK 50 and above, publishing with `webpack` is deprecated.
+
+A web app created using Expo can be served locally for testing out the production behavior. Once the testing phase checks out, you can choose from a variety of third-party services to host it.
+
+| Bundler   | Expo Router | API Routes | Description                                                                                |
+| --------- | ----------- | ---------- | ------------------------------------------------------------------------------------------ |
+| `webpack` | <NoIcon />  | <NoIcon /> | Outputs a Single Page Application (SPA) with a single **index.html** in the output folder. |
+
+## Create a build
+
+Creating a build of the project is the first step to publishing a web app. Whether you want to serve it locally or host it on a third-party service, you'll need to export all JavaScript and assets of a project. This is known as a static bundle. It can be exported by running the following command:
```

Comment:
```suggestion
Creating a build of the project is the first step to publishing a web app. Whether you want to serve it locally or host it on a third-party service, you have to export all JavaScript and assets of a project. This is known as a static bundle. It can be exported by running the following command:
```

- Let's avoid contractions.